### ISSUE TITLE
Feedback about Adafactor

### ISSUE THREAD

**Issue by Fasmanuel2002** [pytorch/pytorch#184668]
There is the following issue on this page: https://docs.pytorch.org/docs/2.12/generated/torch.optim.Adafactor.html

Very good explanation of adafactor in Pytorch documentation, however there is a points I want to declare,

- First: it would be nice if there is anyway that users can see inputs variables trough the web page because sometimes you can forget and don't remember and you need to scroll up to remember and you lose the lecture 

-Second: Maybe I'm wrong but the γ(learning rate) sometimes or is γ or is lr

<img width="574" height="612" alt="Image" src="https://github.com/user-attachments/assets/4d470591-bae7-4379-8d3a-d37f7a71ce48" />

cc @svekars @sekyondaMeta @AlannaBurke @vincentqb @jbschlosser @albanD @janeyx99 @crcrpar

**Comment by janeyx99** [comment]
I don't have a trivial way of solving the first--would take suggestions.

I'd accept a PR using the greek letter throughout for LR.

**Comment by santihdzs** [comment]
I'd like to work on this, will submit a PR shortly.
